src/controllers/activegate/reconciler/statefulset/container_eec.go

```diff
@@ -21,8 +21,6 @@ const extensionsRuntimeDir = "/var/lib/dynatrace/remotepluginmodule/agent/conf/r
 const activeGateInternalCommunicationPort = 9999
 
 const (
-	eecAuthToken = "auth-tokens"
```

Comment:
why is it removed?
EDIT: ok, I see, now it's `ActiveGateVolumeNameGatewayConfig`